### ISSUE TITLE
Autocomplete Keys for get method

### ISSUE THREAD

**Issue by JakobDev** [microsoft/pylance-release#3994]
Consider the following example code:
```python
from typing import TypedDict


class TestType(TypedDict):
    A: str
    B: int


var: TestType = {}
```

Now start typing `var[`. A autocompletion pop-up will appear which has all possible keys from the TypedDict (`A` and `B` in this case) which is very useful. Now start typing `var.get(`. No pop-up will appear. It would be nice, if the Autocompletion would also work with the get method.

**Comment by erictraut** [comment]
You should see a "signature help" popup in this case. This is what VS Code displays when you are completing the arguments to a call expression.

<img width="296" alt="image" src="https://user-images.githubusercontent.com/7040122/220952409-68958216-5646-40ea-bec3-7588b0ad4f37.png">

**Comment by JakobDev** [comment]
I know that, but that's not the Kind of autocompletion I asked for. I asked for this:
![grafik](https://user-images.githubusercontent.com/15185051/220954273-a4ab46f2-863e-4ed4-99b3-76b6a8651496.png)
If you are working a lot with TypedDicts, autocompletion of the Keys is very useful.

**Comment by rchiodo** [comment]
This should already work if you hit CTRL+SPACE

![image](https://user-images.githubusercontent.com/19672699/221000500-3ad40902-e1cf-40c4-b470-4302949eccf9.png)

**Comment by heejaechang** [comment]
this is because we don't do auto trigger for completion this case.

**Comment by heejaechang** [comment]
we still won't show completion when type '(' since we don't want to make that as trigger char, but we will show literals when you type `"` or `'`

**Comment by debonte** [comment]
This issue has been fixed in prerelease version 2023.3.21, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023321-8-March-2023-PreRelease)
